Bottom: 98755dd20c0103988a0feea778739a2cf4ab626a
Top:    5dd6b2e7f2e31388652adf1032557f0c0509d375
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 19:20:06 +0100

TMP TEST


---

diff --git a/src/draw.c b/src/draw.c
index 994dd1c..d8f45bd 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1169,9 +1169,11 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
       if (layernum < max_copper_layer && Layer->On)
         DrawLayerCommon (Layer, drawn_area, true);
 
+#if 0
       /* HACK: Let GTK/GL subcomposite each layer in the group separately */
       if (gui->gui)
         gui->set_layer (0, group, 0);
+#endif
     }
   if (n_entries > 1)
     rv = 1;
